spaces/S000211/README.md

```diff
@@ -0,0 +1,17 @@
+---
+uid: S000211
+name: Join of cofinite and modified right "closed-ray" topologies on $\omega_1+1$
+refs:
+  - doi: 10.1090/S0002-9939-1987-0891170-3
+    name: A distinguishing example in k-spaces (Isbell)
+  - mathse: 3904987
+    name: Answer to "Is there a locally compact space which is not a k-space"
+---
+
+The set $\omega_1+1$ with the topology generated by the basis
+$\{[\alpha,\omega_1]\setminus F:\alpha<\omega_1,F\text{ finite}\}$.
```

Comment:
```suggestion
The set $X=\omega_1+1$ with the topology generated by the base
$\{[\alpha,\omega_1]\setminus F:\alpha<\omega_1,F \text{ finite subset of }X\}$.
The topology is the join in the [lattice of topologies](https://en.wikipedia.org/wiki/Lattice_of_topologies) on $X$ of
(1) the cofinite topology and
(2) the topology with base consisting of the rays $[\alpha,\omega_1]$ for all $\alpha<\omega_1$ (but not $\alpha=\omega_1$).

See {{mathse:5017415}}.
```
extra sentence to explain the name of the space.  Also a link to the specific mathse answer describing the space.